metaseq/checkpoint_utils.py

```diff
@@ -179,14 +179,41 @@ def _delete_old_checkpoint_files(
                 os.remove(old_chk)
 
 
+def verify_shards(cfg, dir=None, checkpoint_name=None):
+    # verifies that all the shards of the checkpoint are present
+    checkpoint_name = checkpoint_name.replace(".pt", "")
+    num_gpus = int(cfg.save_dir[-1:])
```

Comment:
nit: Checkpoint naming format can be changing. So maybe it's better to read ngpu directly from config (cfg.distributed_world_size).

Reply:
Hey Qing, the checkpoint config file doesn't seem to have a distributed_file_size key
![image](https://user-images.githubusercontent.com/13079247/174911789-c8701c92-2793-4231-9c12-4aad7af71f02.png)